ParisBNF/abb/BNFabb94.xml

```diff
@@ -0,0 +1,324 @@
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb94" xml:lang="en" type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Gadla ʾAbunāfǝr, Gadla Kiros, homilies on Melchizedek, Dǝrsana ʾAfnin</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MKr"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <editionStmt>
+                <p/>
+            </editionStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas:  Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>d'Abbadie</collection>
+                        <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b52520186b">BnF Éthiopien d'Abbadie 94</idno>
+                          </msIdentifier>
+
+                          <physDesc>
+                              <objectDesc form="Codex">
+                                  <supportDesc>
+                                      <support>
+                                 <material key="parchment">Parchment</material>
+                                         </support>
+                                      <extent>
+                                          <measure unit="leaf">62</measure>
+                                          <measure unit="leaf" type="blank"/>
+                                          <locus target="#1r #1v #32v #33r #33v #55v #56r #63r #63v"/>
+                                          <dimensions type="outer" unit="mm">
+                                            <height>210</height>
+                                            <width>150</width>
+                                          </dimensions>
+                                      </extent>
+
+                                      <foliation>European foliation in the upper right corner of the folia.
+                                        </foliation>
+
+<collation>
```

Comment:
q1 counts 2 leaves, i and 1, but for the rest of the collation, I think it is safer (and correct) to say that quire structure can't be reconstructed based on the images (you can see how we put it for d'Abbadie 87, for example). I will try to check whether it can be reconstructed at all, once I check it in Paris.